lib/ui/blocs/navigator/navigator_event.dart

```diff
@@ -100,3 +107,7 @@ class NavigateToUserDetails extends NavigatorEvent {
   @override
   List<Object> get props => [user];
 }
+
+class NavigateToShowMnemonicVisualiser extends NavigatorEvent {}
+
+class NavigateToConfirmMnemonicBackupPhrase extends NavigatorEvent {}
```

Comment:
Can you add the docs for these classes? I can't understand how they are used and when they are emitted